### DISCUSSION TITLE
Flet core application properties altered

### DISCUSSION THREAD

**Discussion by lawson-tanner** [flet-dev/flet#2220] [Q&A]
I'm creating an application for MacOS, and noticed when running my code for a different application that it adopted the name, icon and copyright field of a different application, even though it was run from a separate virtual environment. 

I did some poking around in Activity Monitor to look for any clues and found a hidden folder at the user root directory called .flet with a folder for each version which had been installed.

![Screen Shot 2023-12-11 at 3.20.49 am.png](https://github.com/flet-dev/flet/assets/107106831/8dd092b1-54e7-43df-8951-3a6311b37977)

![Screen Shot 2023-12-11 at 3.20.37 am.png](https://github.com/flet-dev/flet/assets/107106831/8b34b988-c466-407f-9b38-4b6d4438dc04)

Note how the bundling of my other application seems to have altered the "core" Flet application instance for the most recent version.

How is this possible given the two projects were run from different virtual environments? Why are they in the user folder?

**Comment by vindevoy** [comment]
I noticed that if you run the application in a browser, it downloads things using PyPi like the flet daemon.  Which caused a problem because I ran it on an internal server with no Internet access.
